src/Compilers/CSharp/Portable/BoundTree/BoundExpressionExtensions.cs

```diff
@@ -40,7 +40,7 @@ public static bool IsLiteralNull(this BoundExpression node)
 
         public static bool IsLiteralDefault(this BoundExpression node)
         {
-            return node.Kind == BoundKind.DefaultExpression && node.Syntax.Kind() == SyntaxKind.DefaultLiteralExpression;
+            return (node.Kind == BoundKind.DefaultExpression || node.Kind == BoundKind.ConvertedDefaultExpression) && node.Syntax.Kind() == SyntaxKind.DefaultLiteralExpression;
```

Comment:
is there a case where the LHS of the && disagrees with the RHS?

Reply:
Good point. I think this could be simplified to just the syntax check.

---
In reply to: [313659786](https://github.com/dotnet/roslyn/pull/37596#discussion_r313659786) [](ancestors = 313659786)